agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -287,20 +324,6 @@ public AzureMonitorTraceExporter buildTraceExporter() {
     } else {
       telemetryItemExporter = new TelemetryItemExporter(pipeline, TelemetryPipelineListener.noop());
     }
```

Comment:
I think we want to create new `TelemetryItemExporter` on each call to `build*Exporter` so that we always get the latest config values

Reply:
it's creating new each time `initExporterBuilder()` is called.